.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: node_js
+node_js:
+  - '12.18.3'
+cache:
+  npm: true
+  directories:
+    - $HOME/.cache/pre-commit
+before_script:
+  - pyenv global 3.7.1
+  - pip3 install pre-commit
+script:
+  - pre-commit run -a
+# To uncomment when we have meaningful code
+# - npm install --log-level=error
+# - npm run lint
```

Comment:
I'd say it wouldn't hurt to uncomment this now to make sure we don't forget to activate these checks. I actually caught a few linting errors that I'll fix real quick with a new commit.

Reply:
I actually also ran lint-fix and it did auto-fix some code, but there was one that had to be manually changed, but I was unsure of how to approach that due to my inexperience. Feel free to push code so that `npm run lint` works though!